### DISCUSSION TITLE
The manual of eventmanager is outdated or incorrect ?

### DISCUSSION THREAD

**Discussion by StefanSa** [drakkan/sftpgo#1191] [Q&A] [answered]
Hi Nicola,
I'm currently trying to rebuild the `recycle bin` following your instructions for the eventmanager, but I'm running into these problems.
In the last 2.4.4 version i use, you can't select `pre-delete` (doesn't exist to select) and `Execute sync` (generates error message).
Tried it anyway, see this error message in the logfile.

` err: action \"move to recycle\" failed: fs rename failed for users: [test]"`

Is this related to this ?
Grazie per il vostro aiuto e il vostro tempo.

**Comment by drakkan** [comment, marked as answer]
You are referring to the docs for the development version. Check the [docs for v2.4.x](https://github.com/drakkan/sftpgo/blob/2.4.x/docs/eventmanager.md). Pre-* actions are not available in any tagged version
